dbt/include/teradata/macros/materializations/incremental/strategies.sql

```diff
@@ -0,0 +1,25 @@
+
+{% macro teradata__get_incremental_default_sql(arg_dict) %}

```

Comment:
Where this macro using in our adapter? Is it by core?

Reply:
We currently not using this, but DBT has mentioned in migration document to maintain this macro.
Maybe it will helpful in upcoming releases.

Reply:
Okay